src/Void.Core/BufferList.fs

```diff
@@ -7,29 +7,23 @@ module Buffer =
         { Filepath = None; Contents = []; CursorPosition = originCell }
 
     let newFile path =
-        BufferType.File { Filepath = Some path; Contents = []; CursorPosition = originCell }
+        { Filepath = Some path; Contents = []; CursorPosition = originCell }
 
     let existingFile path contents =
-        BufferType.File { Filepath = Some path; Contents = contents; CursorPosition = originCell }
+        { Filepath = Some path; Contents = contents; CursorPosition = originCell }
 
-    let empty =
-        BufferType.File emptyFile
-
-    let readLines buffer start =
-        match buffer with
-        | BufferType.File fileBuffer ->
-            fileBuffer.Contents |> Seq.skip (start - 1) // Line numbers start at 1
-        | _ -> Seq.empty
+    let readLines fileBuffer start =
+        fileBuffer.Contents |> Seq.skip (start - 1) // Line numbers start at 1

```

Comment:
`skip`... Interesting. Other languages tend to say `drop`. I guess `skip` fits better with the vocabulary of a sequence.